web/lib/ozoneframework/php/core/ParameterList.php

```diff
@@ -36,46 +36,47 @@ public function initParameterList($runData) {
 			$qs =  $_SERVER['QUERY_STRING'];
 			/* Check if there is a "?" char - if so, remove it. */
 			$qs = preg_replace('/\?.*$/', '', $qs);
-			$splited = explode("/",$qs);
+			$splited = explode('/',$qs);
 			if(count($splited)>= 1){
 				$this->parameterArray['template'] = $splited[0];
-				$this->parameterTypes['template'] = "GET";
-			}
-
-			/* Additionally parse the usual GET parameters. */
-			$uri = $_SERVER['REQUEST_URI'];
-			$uri = preg_replace('/^[^\?]*\?/', '', $uri);
-			$uriPairs = explode('&', $uri);
-			foreach($uriPairs as $uriPair){
-				$u = explode('=', $uriPair);
-				$key = $u[0];
-				$value = $u[1];
-				$this->parameterArray[$key] = urldecode($value);
-				$this->parameterTypes[$key] = "GET";
-				$this->parameterFrom[$key] = 0;
-				$this->allParameters['GET'][$key] = urldecode($value);
+				$this->parameterTypes['template'] = 'GET';
 			}
 
+            /**
+             * If an & is present in the URI, split that into key-value pairs.
+             */
+                $uri = $_SERVER['REQUEST_URI'];
+                $uri = preg_replace('/^[^?]*\?/', '', $uri);
+                $uriPairs = explode('&', $uri);
+                foreach ($uriPairs as $uriPair) {
+                    $u = explode('=', $uriPair);
+                    $key = $u[0];
+                    $value = $u[1];
+                    $this->parameterArray[$key] = urldecode($value);
+                    $this->parameterTypes[$key] = 'GET';
+                    $this->parameterFrom[$key] = 0;
+                    $this->allParameters['GET'][$key] = urldecode($value);
+                }
```

Comment:
This section appears extra indented for some reason

Reply:
It actually is as intended, the diff view doesn't show quite how deep into nested statements that bit is.